### PR TITLE
Makes the lavaland pressure check a proc

### PR DESCRIPTION
**Pull request by ChangelingRain**
I keep wanting to do hot high pressure lavaland but I can't make a heat warning show up for miners under reasonable conditions since their temperature protection is ridiculously OP good
So you get the code improvements from each attempt

### CODE REVIEW

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -553,3 +553,11 @@
 
 /proc/GetBluePart(const/hexa)
 	return hex2num(copytext(hexa, 6, 8))
+
+/proc/lavaland_equipment_pressure_check(turf/T)
+	if(!istype(T))
+		return
+	var/datum/gas_mixture/environment = T.return_air()
+	var/pressure = environment.return_pressure()
+	if(pressure <= 50)
```
Comment (lzimann) [inline code/__HELPERS/game.dm]: 50 should be a define instead

Comment (Cyberboss) [inline code/__HELPERS/game.dm]: 50 should be a define instead

**Review by Cyberboss — DISMISSED** [review]

**Comment by KorPhaeron** [comment]
@ChangelingRain 

Are you planning on making lavaland unlivable without a suit or are you keeping it within human tolerances

**Comment by ChangelingRain** [comment]
I mean I'd want to keep it inside human tolerances but with how temperature protection works that'd massively nerf miner's resistance to fire and lava, and we can't have that.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**